phasmophobia_evidence_v3/widget.css

```diff
@@ -142,7 +146,7 @@
 }
 
 .animated-box.in:after {
-  animation: frame-enter 1s forwards ease-in-out reverse, gradient-animation 4s ease-in-out infinite;
+  animation: frame-enter .5s forwards ease-in-out reverse, gradient-animation 5s ease-in-out infinite;
```

Comment:
Why are you reducing the frame animation's default value to a half second on enter and adding a full second to the gradient?

Reply:
This was actually a "For Me" edit that I completely forgot about and will need to regress.